Updated content for new extension features

japere opened the pull request:
We are releasing a new update for the extension which includes new UI. The doc now includes steps for using the new UI and updated browser support listings. Target for changes to go live 1/10/17-1/12/17. Will update with a comment on the specific date for the doc to go live.

azurecla commented:
Hi __@TwoUnder__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (mausher). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rmca14 commented:
@barbkess 
Please review and approve/disapprove with a comment here, thanks.

JennieHubbard commented:
#sign-off.

japere committed:
Updated content for new extension features

We are releasing a new update for the extension which includes new UI. The doc now includes steps for using the new UI and updated browser support listings. Target for changes to go live 1/10/17-1/12/17. Will update with a comment on the specific date for the doc to go live.

japere committed:
Update publication date
